eclipse-scout-core/src/table/Table.ts

```diff
@@ -3411,6 +3439,16 @@ export class Table extends Widget implements TableModel {
         followingRowSelected = false;
       }
 
+      aria.selected(row.$row, thisRowSelected);
+      row.$row.children('.table-cell').each((index, cell) => {
+        aria.selected($(cell), thisRowSelected);
+      });
+
+      // if single row is selected make it the active descendant for screen readers
+      if (singleRowSelected && thisRowSelected) {
+        aria.linkElementWithActiveDescendant(this.$container, row.$row);
```

Comment:
Active-descendant logic was moved to keystrokes/mousehandler to correctly reflect the active row. It is correct that it only points to a single row on multi-select - however until now there were cases where it was pointing to the wrong row.

I did not implement removal on ctrl-click because it seemed wrong. The active descendant is the current action row, regardless if it is selected or not, because any subsequent keyboard navigation will start there. E.g. if 3 rows are selected and the middle one is ctrl+clicked and 2 remain selected. Is there realy no active descendant? Keyboard navigation will start at the unselected row. Further, low-vision user may profit from the unselected row being announced. I also can open the context menu on that unselected row -> I think we should stick with not removing active-descendant.